content/applications/inventory_and_mrp/inventory/warehouses_storage/replenishment/reordering_rules.rst

```diff
@@ -49,93 +83,93 @@ they sell the product for, so that Odoo knows which company the product should b
+   :icon:`oi-settings-adjust` :guilabel:`(adjust)` icon in the far-right corner, and selecting the
+   desired column from the drop-down menu.
 
 .. _inventory/product_management/trigger:
 
 Trigger
 =======
 
-When stock falls below the reordering rule's minimum, set the reordering rule's *trigger* to
-*automatic* to automatically create purchase or manufacturing orders to replenish stock.
-
-Alternatively, setting the reordering rule's trigger to *manual* displays the product and forecasted
-stock on the *replenishment dashboard*, where the procurement manager can review the stock levels,
-lead times, and forecasted dates of arrival.
-
-.. seealso::
-   :doc:`../replenishment`
-
-.. tip::
-   The :guilabel:`Replenishment` dashboard is accessible by going to :menuselection:`Inventory app
-   --> Operations --> Replenishment`.
+A reordering rule's *trigger* can be set to *automatic* or *manual*. While both function the same
+way, the difference between the two types of reordering rules is how the rule is launched:
 
-To enable the :guilabel:`Trigger` field, go to :menuselection:`Inventory app --> Configuration -->
-Reordering Rules`. Then, click the :guilabel:`(slider)` icon, located to the far-right of the column
-titles, and enable the :guilabel:`Trigger` option from the additional options drop-down menu that
-appears.
+- :ref:`Auto <inventory/warehouses_storage/auto-rr>`: a purchase or manufacturing is automatically
+  created when the forecasted stock falls below the reordering rule's minimum. By default, the
+  :guilabel:`Auto` trigger is selected.
+- :ref:`Manual <inventory/warehouses_storage/manual-rr>`: receive a suggestion on the
+  :doc:`Replenishment report <report>`, so the user can review the stock levels, lead times, and
+  forecasted dates of arrival, before clicking the *Order Once* button.
 
-.. image:: reordering_rules/enable-trigger.png
-   :align: center
-   :alt: Enable the Trigger field by toggling it in the additional options menu.
+To enable the :guilabel:`Trigger` field, go to :menuselection:`Inventory app --> Operations -->
+Replenishment` or :menuselection:`Inventory app --> Configuration --> Reordering Rules`. Then, click
+the :icon:`oi-settings-adjust` :guilabel:`(adjust)` icon, located to the far-right of the column
+titles, and tick the :guilabel:`Trigger` checkbox.
 
 In the :guilabel:`Trigger` column, select :guilabel:`Auto` or :guilabel:`Manual`. Refer to the
 sections below to learn about the different types of reordering rules.
 
+.. _inventory/warehouses_storage/auto-rr:
+
 Auto
 ----
 
 Automatic reordering rules, enabled by setting the reordering rule's :guilabel:`Trigger` field to
 :guilabel:`Auto`, generate purchase or manufacturing orders when:
 
-#. the scheduler runs, and the *On Hand* quantity is below the minimum
+#. the scheduler runs, and the *Forecasted* quantity is below the minimum
 #. a sales order is confirmed, and lowers the *Forecasted* quantity of the product below the minimum
 
 .. tip::
```

Comment:
This is the first time I'm reading about the scheduler. Maybe instead of a `.. tip::`, there can be a separate sub-section to talk about the scheduler?